### PR TITLE
Add some portuguese labels in labels.json

### PR DESCRIPTION
**Pull request by InsertyEXE**

### CODE REVIEW

File: labels.json [hunk 1]
```diff
@@ -1561,6 +1571,11 @@
                 "lang": "en",
                 "name": "Raptor",
                 "description": "This user is a Raptor! [Category: Dinosaur]"
+            },
+             {
+                "lang": "pt-BR",
+                "name": "Velociraptor",
+                "description": "Este usuário é um Velociraptor! [Categoria: Dinossauro]"
```
Comment (astrabun) [inline labels.json]: Please revert "Categoria" to "Category" (this is used on the sonasky-browse site for parsing; Dinossauro is fine, but "Category" must remain.

File: labels.json [hunk 2]
```diff
@@ -1591,6 +1616,11 @@
                 "lang": "en",
                 "name": "Eagle",
                 "description": "This user is an Eagle! [Category: Avian/Bird]"
+            },
+            {
+                "lang": "pt-BR",
+                "name": "Águia",
+                "description": "Este usuário é uma Águia! [Categoria: Ave/Pássaro]"
```
Comment (astrabun) [inline labels.json]: Same

**Review by astrabun — CHANGES_REQUESTED** [review]
See comments; please keep "Category" untranslated (but the text of the category can remain)!

File: labels.json [hunk 3]
```diff
@@ -1641,6 +1691,11 @@
                 "lang": "en",
                 "name": "Crow",
                 "description": "This user is a Crow! [Category: Avian/Bird]"
+            },
+             {
+                "lang": "pt-BR",
+                "name": "Corvo",
+                "description": "Este usuário é um Corvo!"
```
Comment (astrabun) [inline labels.json]: Please do not omit "Category" when it is in the other localization :)

File: labels.json (hunk 1 above)
Comment (InsertyEXE) [reply labels.json]: Okay! I'll revert it

**Comment by InsertyEXE** [comment]
I think it's okay this time

**Review by astrabun — APPROVED** [review]
Looks great! Thanks so much both for contributing and for the updates <3

**Pull request merged**